tests/ResolverTests.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Kettle Config Loader Tests
+ *
+ * Copyright 2013 OCAD University
+ * Copyright 2012-2015 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/fluid-project/kettle/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.require("%kettle"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit");
+
+kettle.loadTestingSupport();
+
+fluid.registerNamespace("kettle.tests");
+
+kettle.tests.loadConfigTest = function (configName) {
+    return kettle.config.loadConfig({
+        configName: configName,
+        configPath: "%kettle/tests/configs"
+    });
+};
+
+jqUnit.test("Test present environment substitution", function () {
```

Comment:
Shall the case that the requested global env variable is not defined at all be tested?